testplan/common/serialization/fields.py

```diff
@@ -283,7 +283,7 @@ def __init__(
         self.schema_context = schema_context
         self.type_field = type_field
         self.many = kwargs.get("many", False)
-        super(GenericNested, self).__init__(default=default, **kwargs)
+        super(GenericNested, self).__init__(default=default, metadata=kwargs)
```

Comment:
this change is to address `RemovedInMarshmallow4Warning: Passing field metadata as keyword arguments is deprecated. Use the explicit `metadata=...` argument instead. Additional metadata: `